packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5135,6 +5298,33 @@ Client authentication credentials for an [OAuth 2.0 Authorization Server (AS)](h
 }
 ```
 
+##### Apple Client Signing object
+
+The information is used to generate the secret JSON Web Token for the token requests to Apple IdP.
+
+| Property      | Description                                                                                                 | DataType | Nullable | Readonly | MinLength | MaxLength |
+| ------------- | ----------------------------------------------------------------------------------------------------------- | -------- | -------- | -------- | --------- | --------- |
+| privateKey    | The  PKCS #8 encoded private key that you created for the client and downloaded from Apple                  | String   | TRUE     | FALSE    | 1         | 1024      |
+| kid           | The Key ID that you obtained from Apple when you created the private key for the client                     | String   | FALSE    | FALSE    | 1         | 1024      |
+| teamId        | The Team ID associated with your Apple developer account                                                    | String   | FALSE    | FALSE    | 1         | 1024      |
+
+> **Note:** privateKey is required in the CREATE request, but it is nullable when performing an UPDATE and keeps the existing value if it is null. It will not be returned for all the LIST and GET requests, as well as the UPDATE requests with it is null.
```

Comment:
`privateKey` is required for a CREATE request. For an UPDATE request, it can be null and keeps the existing value if it is null. `privateKey` isn't returned for LIST and GET requests or UDPATE requests if it is null.